### PR TITLE
Add a "Become a Partner" button onto the Partners Index Page.

### PR DESCRIPTION
**Pull request by kylorhall**
[Vimaly Story](https://vimaly.com/#j8mqm/t/www_Add_Become_a_Partner_to_the_Partner_Directory./YZcfYboXUkfCE2LTT)

Screenshot:
Screen Shot 2021-01-19 at 1 45 40 PM

### CODE REVIEW

File: source/css/partners/components/_header.scss [hunk 1]
```diff
@@ -29,18 +35,15 @@ $__categories_list_bottom_margin: $gutter/2;
     }
 
     &-list {
-      margin-left: $gutter / 3;
+      // margin-left: $gutter / 3;
```
Comment (maisiembleton) [inline source/css/partners/components/_header.scss:38]: how come this is left in but commented out?

File: source/css/partners/components/_header.scss [hunk 2]
```diff
@@ -29,18 +35,15 @@ $__categories_list_bottom_margin: $gutter/2;
...
     }
   }
 
   &__category {
-    display: inline-block;
-    margin: 0 $gutter/4 $__categories_list_bottom_margin;
-    padding: 0 $gutter/2;
-    border-radius: $gutter / 4;
+    @extend %partners-pill-btn;
```
Comment (maisiembleton) [inline source/css/partners/components/_header.scss:43]: this is cool

File: spec/features/partner_categories_page_spec.rb [hunk 3]
```diff
@@ -86,6 +86,9 @@
           )
         end
 
+        # Have a "Become a Partner" button.
+        expect(page).to have_css("a[href='#{localize_url('become-a-partner', locale_id: locale[:id])}']", text: 'Become a Partner')
```
Comment (maisiembleton) [inline spec/features/partner_categories_page_spec.rb:90]: so interesting that there's a space between `to` and `have_css` - different syntax I guess. Good to see how this is done in ruby though!

**Review by maisiembleton — APPROVED** [review]
Looks good! Just a thought - do you reckon the 'Become a Partner' button might blend in with the other Jump To links - being grouped close and same shape/size? 
Haven't seen scss in a while! Thanks for pinging me on this - good to have a small refresher on styling outside of styled-components

Screen Shot 2021-01-19 at 3 11 26 PM
Screen Shot 2021-01-19 at 3 12 14 PM

File: spec/features/partner_categories_page_spec.rb (hunk 3 above)
Comment (kylorhall) [reply spec/features/partner_categories_page_spec.rb:90]: This is:
```ruby
expect(page).to(have_css('…'))
```

Just in Ruby, unless you do a few specific things or chain off of it, the `()` are optional.

File: source/css/partners/components/_header.scss (hunk 1 above)
Comment (kylorhall) [reply source/css/partners/components/_header.scss:38]: Ah, I will just remove actually.

**Comment by kylorhall** [comment]
@maisiembleton Mostly just the responsiveness and alignment of making it "inline" with `Partners` (and the other dozen category titles this could be) takes extra work.  I'm just aiming for a 5 minute PR here, I think it fits in fine as that, but also…could spend more to do a slightly better job.  Might pass your feedback by Ben to see if he has any feedback as well…because it is moving the content lower on the page quite a bit.

**Pull request merged**